zbuf/combiner.go

```diff
@@ -4,25 +4,45 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/brimsec/zq/pkg/nano"
 	"github.com/brimsec/zq/zng"
 )
 
+type Direction bool
```

Comment:
@mattnibs : I know an early version of your Storage PR had a similar direction option for Combiner, but I didn't have a copy of that, so not sure how this compares with your implementation, PTAL.

Reply:
My version used expr.SortFn which IMO makes this more flexible. You can see my pass for Combiner here: https://github.com/brimsec/zq/pull/823/files#diff-e97e96d1cbbd75b1be023a168075c8ec

Reply:
updated in 5dd8d23